OSDOCS-16573: Fixed Vale warnings in ROSA CLI docs (#110749)
openshift/openshift-docs: cli_reference/rosa_cli/rosa-get-started-cli.adoc

:context: rosa-getting-started-cli
toc::[]

[role="_abstract"]

> Use this guide to learn about the {rosa-cli-first}, install it, configure it, and keep it up to date.

Reviewer: ```suggestion
Use this guide to learn how to install, configure, and update the {rosa-cli-first}.
```
?
